src/components/Calculator.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import './Calculator.css';
+
+export default function Calculator() {
+  return (
+    <div className="text-[#333] text-sm font-bold p-12">
+      <table className="h-[300px] w-[350px] mx-auto bg-[#E0E0E0]">
+        <thead>
+          <tr>
+            <td colSpan="4">
+              <input type="text" value="0" className="w-full pr-1 text-right text-xl font-normal text-white bg-[#858693]" />
+            </td>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              <input type="button" value="AC" />
+            </td>
+            <td>
+              <input type="button" value="+/-" />
+            </td>
```

Comment:
- [ ] [OPTIONAL] your calculator looks nice but it would be better if `Calculator.js` components have at least one child component in order so that it will be reuseable through the app 
- Screenshot of the requirements 
<img width="315" alt="Screenshot 2023-05-02 at 23 19 12" src="https://user-images.githubusercontent.com/61790587/235798229-fba3b646-fc63-43a6-a163-4ec6e9d2df4c.png">

Reply:
thank you for your comment